src/main/kotlin/nl/avisi/structurizr/site/generatr/site/views/Page.kt

```diff
@@ -22,6 +22,10 @@ private fun HTML.headFragment(viewModel: PageViewModel) {
             rel = "stylesheet",
             href = "../" + "/style.css".asUrlToFile(viewModel.url)
         )
+        link(
+            rel = "stylesheet",
+            href = "../" + "/style-branding.css".asUrlToFile(viewModel.url)
```

Comment:
`"../" + "/style-branding.css"` will result in `"..//style-branding.css"`. Is that intentional?

Reply:
That is not the case here, the `.asUrlToFile(viewModel.url)` results in an url starting without a `/`.